src/config/featureConfiguration.json

```diff
@@ -0,0 +1,58 @@
+[
+  {
+    "index": 0,
+    "internalName": "TASK_STATUS_MENU",
+    "displayName": "Task Status Menu",
+    "description": "Enables a right click menu for each task to allow you to select the task Status from the available next transition states.",
+    "enabledByDefault": false,
+    "stable": false
+  },
+  {
```

Comment:
Can you find a way to not add these strings to Tasks please?

When people see this in the Tasks code, it is going to cause a lot of confusion as they will expect the features to be provided.

In general, I will be seeking to have each new feature be made release-quality before it is added to Tasks, so am unsure whether the Tasks library needs this feature?

Reply:
User testing in a simple way via features allows better feedback without impacting a larger user base, feature flags are a common way to do this. 

This allows the changes to move through faster and not impact users immediately but let them try it out if they want and when not issues are found the feature can move to being on by default.

It also means if there is a problem that is found later on the risk is already mitigated or mitigatable by the user just turning it off again. 

I am trying to contribute back here.

Reply:
Thanks very much indeed for removing the unreleased features - much appreciated.

> I am trying to contribute back here.

I totally understand that, and really appreciate it.